README.md

```diff
@@ -8,6 +8,10 @@ Notes
 ===
 * Heavily cribbed from https://github.com/bluenviron/gortsplib/blob/main/examples/client-read-format-h264-convert-to-jpeg/main.go
 
+FFmpeg
+===
+
+This project is designed to work with FFmpeg version 6.1. During the build process of the module, platform-specific static builds of FFmpeg 6.1 will be created. This ensures that the module is always using the correct version of FFmpeg, regardless of what other versions might be installed on your system.
```

Comment:
Can we please add instructions for how you recommend production builds of the module be produced on linux x86_64, linux arm, and macos arm?

Reply:
Added in instructions for arm64/x86 builds.
- For now, will not be able to have a fully static build for macos
    - Requires libjpegturbo dylib on device (Which seems to be the case for viam-server as well).